tests/appsec/utils.py

```diff
@@ -0,0 +1,12 @@
+import sys
+
+
+class Mock_Alternative:
```

Comment:
nit: I don't love the "Mock_Alternative" name , maybe we call this "Either" or "In" so it would be used as `Either(True, None)` or `In(True, None)`? WDYT?

Reply:
I'm open to a new name. Either is restricted to only two options in english, and In is too short and you're not sure if it starts with an i and an l. Any other suggestions?

Reply:
Well, I went with Either, it looks like the best alternative ;)